src/graphics.zig

```diff
@@ -1176,6 +1177,7 @@ const TextRendering = struct { // MARK: TextRendering
 		try ftError(hbft.FT_Set_Pixel_Sizes(freetypeFace, 0, textureHeight));
 		harfbuzzFace = hbft.hb_ft_face_create_referenced(freetypeFace);
 		harfbuzzFont = hbft.hb_font_create(harfbuzzFace);
+		fontUnitPerPixel = @as(f32, @floatFromInt(freetypeFace.*.units_per_EM))/@as(f32, @floatFromInt(textureHeight));
```

Comment:
I don't think you need to dereference here.
```suggestion
		fontUnitPerPixel = @as(f32, @floatFromInt(freetypeFace.units_per_EM))/@as(f32, @floatFromInt(textureHeight));
```

Reply:
error reported if access without dereferrence:
```
type '[*c]cimport.struct_FT_FaceRec_' does not support field access
```